### DISCUSSION TITLE
如果把.pt训练的模型转换成.torchscript

### DISCUSSION THREAD

**Discussion by yaomoshen520** [ultralytics/ultralytics#6775] [Docs]
如果把.pt训练的模型转换成.torchscript.查看了文档好像没看到

**Comment by pderrenger** [comment]
@yaomoshen520 您好！将 `.pt` 训练的模型转换为 `.torchscript` 格式是完全可行的。在 YOLOv8 中，您可以使用 `export` 模式来执行这个转换。以下是一个简单的示例代码，展示了如何将模型导出为 TorchScript 格式：

```python
from ultralytics import YOLO

# 加载您的预训练模型
model = YOLO('path/to/your/model.pt')

# 导出模型为 TorchScript 格式
model.export(format='torchscript')
```

在执行 `export` 方法时，默认的导出格式就是 `torchscript`，所以您甚至可以省略 `format` 参数。导出后的模型将保存在您模型文件所在的同一目录下。

更多关于导出模型的信息，您可以参考我们的文档中的 [Export](https://docs.ultralytics.com/modes/export) 部分。希望这能帮助到您！ 😊🚀
